db/fixtures/ae_datastore/ManageIQ/ConfigurationManagement/AnsibleTower/Operations/StateMachines/Job.class/__methods__/launch_ansible_job.rb

```diff
@@ -35,13 +35,14 @@ def ansible_vars_from_objects(object, ext_vars)
   end
 
   def object_vars(object, ext_vars)
-    object.attributes.each_with_object(ext_vars) do |(key, value), hash|
+    key_list = object.attributes.keys.select { |k| k.start_with?('param', 'dialog_param') }
```

Comment:
@mkanoor In other words, save yourself from having to remember why you did this...